app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -160,7 +161,7 @@ module.exports = React.createClass
           disabled = @getDisabledAttribute(i)
           selectedOption = if @state.selectedOptions[i]?.value then @state.selectedOptions[i] else null
           optionsKeys.push selectedOption.value if selectedOption?.value
-          <div key={select.id}>
+          <div id={select.id} key={select.id}>
```

Comment:
`.find({key: 'countyID'})` should work. You can't use CSS selectors for React props. The g[uide to Enzyme Selectors](http://airbnb.io/enzyme/docs/api/selector.html) is really useful here.